### PR TITLE
Add duration examples

### LINKED ISSUE #588: Allow numerical seconds for durations.
It might be useful to allow seconds in numerical form. e.g. Instead of "P20S", using "20" might be easier for those generating statement as well as for storage and comparison. 

As ISO8601 has its own benefits, allowing both "P20S" and "20" might be useful.

### PR DESCRIPTION
**Pull request by garemoko**
Fixes #588 

Possibly controversial points:

* The word "approximately" on line 2797
* Lines 2805 and 2806
* The lack of period in the braced sentence on line 2789

@fugu13 as you've been involved in the #588 discussion hopefully this is a quick one for you to review.

### CODE REVIEW

File: xAPI.md [hunk 1]
```diff
@@ -2774,6 +2774,36 @@ be in months or years.
...
+		<td>P3Y1M29DT4H35M59.14S</td>
+		<td>A duration also including years, months and days.</td>
+	</tr>
+	<tr>
+		<td>P3Y</td>
+		<td>Aproximately three years e.g. completion of a qualification.</td>
+	</tr>
+	<tr>
+		<td>P4W</td>
+		<td>Four weeks. Not that weeks cannot be combined with other time peroids. 'P4W1D' is not valid.</td>
```
Comment (garemoko) [inline xAPI.md]: Not = Note 
I'll update with any other changes.

File: xAPI.md [hunk 2]
```diff
@@ -2774,6 +2774,36 @@ be in months or years.
...
+		<td>The same time peroid as above represented in seconds. 
+		(Note: if the time peroid in question contained a leap second, this conversion would be inaccurate)</td>
+	</tr>
+	<tr>
+		<td>P3Y1M29DT4H35M59.14S</td>
+		<td>A duration also including years, months and days.</td>
+	</tr>
+	<tr>
+		<td>P3Y</td>
+		<td>Aproximately three years e.g. completion of a qualification.</td>
```
Comment (garemoko) [inline xAPI.md]: Approximately

File: xAPI.md [hunk 3]
```diff
@@ -2774,6 +2774,36 @@ be in months or years.
...
+	</tr>
+	<tr>
+		<td>P4W</td>
+		<td>Four weeks. Not that weeks cannot be combined with other time peroids. 'P4W1D' is not valid.</td>
+	</tr>
+</table>
+
+Duration are expected to be presented in the format in which they are recorded. For example if a duration is tracked
```
Comment (garemoko) [inline xAPI.md]: Durations (missing s)

**Comment by andyjohnson** [comment]
+1, nicely done!

**Comment by garemoko** [comment]
Can we get another +1 or some comments to progress this forwards?

**Comment by andyjohnson** [comment]
Moving this forward, the examples all look correct with ISO8601

**Pull request merged**